tools/debugger/test/check-order.js

```diff
@@ -55,262 +70,512 @@ describe('Orders', async () => {
     )
 
     const errors = await checker.checkOrder(order, 'rinkeby')
-    assert.equal(errors.length, 0)
+    assert.equal(errors.length, 1)
+    assert.equal(errors[0], 'sender token kind invalid')
   })
 
-  it('Check expired order', async () => {
-    const order = await orders.getOrder({
-      expiry: MAY_11_2017_00_00_00,
-      nonce: '101',
-      signer: {
-        wallet: KNOWN_GANACHE_WALLET,
-        token: ASTAddress,
-        amount: '400',
-      },
-      sender: {
-        wallet: senderWallet,
-        token: WETHAddress,
-        amount: '2',
-      },
+  describe('ERC20 Swaps', async () => {
+    it('Check correct order with signature', async () => {
+      const order = await orders.getOrder({
+        expiry: NOV_7_2020_22_18_14,
```

Comment:
why was the expiry updated to 2020?

Reply:
That's just because of how Github has done the diff. If you find the test `Check expired order` on the right hand side, it still has an expiry of `MAY_11_2017_00_00_00`.